client/my-sites/email/email-providers-in-depth-comparison/data.ts

```diff
@@ -0,0 +1,23 @@
+import { translate } from 'i18n-calypso';
+import { ProviderComparison } from 'calypso/my-sites/email/email-providers-in-depth-comparison/comparison-table/';
+
+// eslint-disable-next-line @typescript-eslint/no-empty-function
+const noop = () => {};
+
+export const professionalEmailComparisonObject: ProviderComparison = {
```

Comment:
Following up on my other comment about naming, I would go with this:

```suggestion
export const professionalEmailFeatures: EmailProviderFeatures = {
```